dbt-tests-adapter/dbt/tests/adapter/incremental/fixtures.py

```diff
@@ -303,3 +303,34 @@
 
 from source_data
 """
+
+_MODELS__SRC_ARTISTS = """
```

Comment:
Nit: The content of the model looks like "jobs" now instead of "artists". So ideally they'd match instead of being different.